server/src/users/EventRouter.ts

```diff
@@ -18,31 +18,113 @@ export const eventRouter = Express.Router();
 
 // GET requests/:id currently using the Name field (first name) since don't know how to actually grab object id in salesforce
 
+eventRouter.get("/", async (req: Express.Request, res: Express.Response) => {
```

Comment:
clean up console.log statements and commented out code

Reply:
I removed the console.log statements in e361081. I didn't see any commented code in this file though :/